roles/junit2json/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: Apache-2.0
+---
+# tasks file main.yml for role redhatci.ocp.junit2json
+
+- name: Validate some variables
+  ansible.builtin.assert:
+    that: "{{ condition }}"
+  loop:
+    - global_reports_path_patterns | length > 0
```

Comment:
This variable should follow the naming convention of the role (be prefixed by `junit2json_` keyword)